css-two-column-layout/styles.css

```diff
@@ -0,0 +1,85 @@
+.container {
+  margin: auto;
+  max-width: 1200px;
+  background-color: white;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.col-full {
+  width: 100%;
+}
+
+.col-half {
+  width: calc(100%/2);
+}
+
+.col-fourth {
+  width: 25%;
+  text-align: right;
+  padding-right: 0;
```

Comment:
`column` classes should only ever have `flex-basis`, nothing else. Anything additional necessary should be on a separate class.